core/src/main/java/org/apache/accumulo/core/client/BatchScanner.java

```diff
@@ -16,19 +16,20 @@
  */
 package org.apache.accumulo.core.client;
 
+import org.apache.accumulo.core.data.Range;
+
 import java.util.Collection;
 import java.util.concurrent.TimeUnit;
 
-import org.apache.accumulo.core.data.Range;
-
 /**
  * Implementations of BatchScanner support efficient lookups of many ranges in accumulo.
+ * BatchScanners are also appropriate for large, single ranges,
+ * as a BatchScanner will break those ranges up into separate RPCs
+ * provided the range spans more than one tablet
+ * and there are sufficiently many scan threads available.
```

Comment:
Maybe instead of suggesting how to use the batch scanner, we could focus more on describing possible behavior?

  * May parallelize reading of data.  Multiple input ranges may be read in parallel or sub ranges of individual input ranges may be read in parallel.
  * May return data in unsorted order.
  * May batch multiple ranges into a single RPC to a tserver.   

Could still mention possible use cases like :

  * Looking up lots of small ranges
  * Parallelizing (spell check in web browser does not like that word) a computation over an entire table using a large range w/ iterators.   This case may not return lots of data, although lots of data may be read by the iterators.